frontend/src/components/ChatBox/ChatBoxMessage/MessageBubble.tsx

```diff
@@ -34,16 +34,33 @@ function MessageBubble({
 		positionClassName
 	);
 
+	const messageAuthor =
+		message.type === CHAT_MESSAGE_TYPE.LEVEL_INFO
+			? 'Information message: '
+			: message.type === CHAT_MESSAGE_TYPE.USER
+			? 'You said: '
+			: message.type === CHAT_MESSAGE_TYPE.USER_TRANSFORMED
+			? 'Your message transformed by XML tagging: '
+			: message.type === CHAT_MESSAGE_TYPE.ERROR_MSG
+			? 'Error message: '
+			: message.type === CHAT_MESSAGE_TYPE.BOT
+			? 'ScottBrewBot said:  '
+			: 'ScottBrewBot said: ';
+
 	return (
-		<div className={className} lang="en">
+		// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+		<section className={className} tabIndex={0}>
 			{message.type === CHAT_MESSAGE_TYPE.USER_TRANSFORMED && (
 				<b>Transformed: </b>
 			)}
 			{message.type === CHAT_MESSAGE_TYPE.LEVEL_INFO && (
 				<p className="level-info-header">Information</p>
 			)}
-			{message.message}
-		</div>
+			<p>
+				<span className="visually-hidden">{messageAuthor}</span>
+				{message.message}
+			</p>
```

Comment:
This `<p>` element is adding too much spacing to the chat bubbles:

![image](https://github.com/ScottLogic/prompt-injection/assets/103250539/01b1737b-82a3-4496-b81f-ed2178edc98f)

Just need to remove the margin.